pkg/kube/config.yaml

```diff
@@ -4,3 +4,6 @@
 write-kubeconfig-mode: "0644"
 cluster-init: true
 log: "/var/lib/rancher/k3s/k3s.log"
```

Comment:
Do we want this to be `debug: true` and `etcd-expose-metrics: true` all of the time?

Reply:
I think longterm we'll want debug disabled, but for now while in QA/testing we want debug enabled.  Metrics we want available longterm, the first user of the etcd metrics will be plumbing the fdatasync time through existing eve metrics for tracking health.

Reply:
Then let's add a comment here, and some tracker somewhere that we remove this soon. Even better would be a flag to turn it on during installation or startup. I am worried about this becoming tech debt that we all forget about, and then spend time in 6 months wondering "where all those logs (that fill up filesystems) are coming from, and... oh yeah, how did we forget about that???"

Reply:
Yes I understand the worry there, I can add a "Remove me" type comment and add a ticket to follow up.